src/components/JournalIndexPage/index.jsx

```diff
@@ -12,7 +13,7 @@ const JournalCard = props => (
   <Link className="journal-card-link" to={{ pathname: props.authorized ? `/${props.journalAboutId}` : `/${props.journalAboutId}/about` }}>
     <div className="journal-card" >
       <div className="journal-card-image">
-        <img src={props.cardImageUrl} alt={props.title} />
+        <LogoRender siteLogo={props.cardImageUrl} defaultImage={BookCard} altText={props.title} />
```

Comment:
Using a `LogoRender` component and a `siteLogo` prop on a non-logo image, seems a bit off. In this case it's acting more as an `ImageWithDefault`-type component. If you want to keep everything else the same, I think changing the component and prop names would help. I'm also not entirely sure why `Img` and `LogoRender` are different components. It looks like `LogoRender` just passes the props down to `Img`. Wondering if they can be combined to simplify.